zetkin/app.zetkin.org
# Shortcut to set current user as campaign manager

## Changes in summary
This PR adds a shortcut under the "Campaign manager" field in the campaign form, to directly select the current user as the manager.

The shortcut is only visible when the user is not already the selected manager, i.e. when there is no manager assigned or when another person is assigned as manager.

To solve this, I had to switch the `Autocomplete` used to select manager to [controlled](https://reactjs.org/docs/forms.html#controlled-components). I think this is a fine solution, but I'm curious to hear the thoughts of others!

Resolves #417

## How to test
1. Navigate to the "All campaigns" page
2. Via the Speed Dial in the bottom right, click to create a new campaign
3. In the "Create new campaign" form, click the new "Select yourself as manager" – "Angela Davis" (or whatever your user is) should become selected and the link disappear
4. Search to select someone else – the link be vissible
5. Use the "x" to select no manager – the link be visible

## Screenshots
![image](https://user-images.githubusercontent.com/550212/135079202-99b7b2a8-591f-49a8-a368-7dd76d904fd8.png)
